.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test
```

Comment:
how about call it as `Linting` or just `Lint` or `Linter` ;)

Reply:
That's what you had on original project, ok I will change it back 😛